ml/NN/config/NN.conf.example

```diff
@@ -0,0 +1,17 @@
+[NN]
+mode = train
+input_file= 8_12_VT_peinfo_objdump_join.parquet
+model = NN_model
+output = predictions
+batch_size = 10
+epochs = 1
+cluster_size = 3
+steps= 1000
+
+# If you want to predict the label of the labeled data.
```

Comment:
Oh, it's my fault. I just copy and forget to change the word
I will change it to this:
```
# If you want to predict the label of the labeled data.
# change the mode:
# mode = inference

# If you want to predict the data without the label.
# change the mode:
# mode = predict
```
Now, this program has three modes: train, inference, and predict. The user can only choose one mode to run the program every time.

Which one do you think better?
train = True; predict = False;  inference= False
OR
mode = train ( predict/inference )